com.unity.formats.fbx/Tests/FbxTests/ModelExporterTest.cs

```diff
@@ -1048,18 +1048,29 @@ public void TestPreserveImportSettings()
             importer.importBlendShapes = false;
             importer.SaveAndReimport();
 
-            // re-export with preserve import settings true and verify guid and settings are the same
+            // save original fbx's guid
+            var originalGuid = AssetDatabase.AssetPathToGUID(filename);
+
```

Comment:
This test will break if the default value for "importBlendShapes" changes in the future.

I think you should "remember" the original value like you do for the guid, then set the opposite value before exporting (!originalImportBlendShape)